config/locales/en.yml

```diff
@@ -3331,6 +3331,8 @@ See the %{link} to find out more about %{sitename}'s features and to start using
           stripe:
             error_saving_payment: Error saving payment
             submitting_payment: Submitting payment...
+          paypal:
+            no_payment_via_admin_backend: No Payment Via Admin Backend
```

Comment:
awesome :+1: 
Shall we improve the message a little?
I am not sure but I think this is "Paypal payments cannot be captured in the Backoffice"

Reply:
yes, I prefer that

Reply:
I'm very happy to update this one! And I have some time today :-D